ducktape/tests/runner.py

```diff
@@ -117,10 +117,10 @@ def run_all_tests(self):
                 result.stop_time = time.time()
                 self.results.append(result)
 
+                self.log(logging.INFO, "Summary: %s" % str(result.summary))
+                self.log(logging.INFO, "Data: %s" % str(result.data))
                 test_reporter = SingleResultFileReporter(result)
                 test_reporter.report()
-                test_reporter = SingleResultStdoutReporter(result)
```

Comment:
At a minimum, there's an extraneous import now. But also, if we're not even creating this anymore, do we need to keep the class around at all?